### PR TITLE
Add default command

### LINKED ISSUE #106: Add command to set default platform
A `default` command should be added so that the user may change the default platform for ponyup from a list of options. This will be used by the init script to have the user select rather than trying to guess things like libc implementation.

### PR DESCRIPTION
**Pull request by Theodus**
Closes #106

### CODE REVIEW

File: ponyup-init.sh [hunk 1]
```diff
@@ -84,47 +83,29 @@ esac
...
-        break
-        ;;
-      3 | cancel)
-        exit 1
-        ;;
-      *)
-        ;;
-      esac
-    done
+  *) ;;
```
Comment (SeanTAllen) [inline ponyup-init.sh:96]: Should this prompt the user to run the default command?

**Comment by SeanTAllen** [comment]
@Theodus This isn't documented in the README. How would someone if they hit the problem like there is on Fedora know to run `default`?

**Comment by Theodus** [comment]
I expect that it shouldn't occur, since the init script should catch that and prompt the user.

**Comment by SeanTAllen** [comment]
How? It looks like it was prompting before and you simply moved it to later in the script.

**Comment by Theodus** [comment]
I'm not sure why it wouldn't have worked previously. I'll add a section in the readme as well, but we should investigate with someone who has Fedora.

**Comment by SeanTAllen** [comment]
Sounds good. Perhaps we can spin up a docker container with Fedora in it.

**Comment by Theodus** [comment]
I spun up a docker container and found the problem. The issue was that piping into sh doesn't work as expected.

**Comment by SeanTAllen** [comment]
@Theodus are there updates coming to this PR based on README comments from earlier + what you find with piping to sh?

**Comment by Theodus** [comment]
Sorry, I forgot to push the readme update. I've done that now.

Piping the script into sh results in the platform select prompt to be non-interactive and loop until sh decides to bail out and exit.

**Comment by SeanTAllen** [comment]
Sounds like we should change instructions to be download and run. 

Thoughts?

**Comment by Theodus** [comment]
The readme was updated to use the following:
```
sh -c "$(curl --proto '=https' --tlsv1.2 -sSf https://raw.githubusercontent.com/ponylang/ponyup/latest-release/ponyup-init.sh)"
```
which works on Fedora.

**Pull request merged**